api/services/emAnno.ts

```diff
@@ -27,46 +27,49 @@ export async function deleteAnnotation(uuid: string) {
     return request.data
 }
 
-export default app => {
-    const router = express.Router({ mergeParams: true })
-    app.use('/em-anno', router)
-
-    router.post('/annotation-sets', async (req, res, next) => {
+export async function handlePost(req: Request, res: Response): Promise<void> {
+    {
         // Called when get annotation-sets returns 404
         const response = await asyncReturnOrError(createAnnotationSet(req.body), ' Error creating annotations', res, logger)
 
         if (response) {
             res.status(200).send(JSON.stringify(response))
         }
-    })
-
-    router.get('/annotation-sets/:uuid', async (req, res, next) => {
-        const uuid = req.params.uuid
-        const response = await asyncReturnOrError(getAnnotionSet(uuid), ' Error getting annotations', res, logger)
-
-        if (response) {
-            res.status(200).send(JSON.stringify(response))
-        }
+    }
+}
 
-    })
+export async function handleGet(req: Request, res: Response): Promise<void> {
+    const uuid = req.params.uuid
+    const response = await asyncReturnOrError(getAnnotionSet(uuid), ' Error getting annotations', res, logger)
 
-    router.delete('/annotations/:uuid', async (req, res, next) => {
-        const uuid = req.params.uuid
+    if (response) {
+        res.status(200).send(JSON.stringify(response))
+    }
+}
 
-        const response = await asyncReturnOrError(deleteAnnotation(uuid), ' Error deleting annotations', res, logger)
+export async function handleDelete(req: Request, res: Response): Promise<void> {
+    const uuid = req.params.uuid
+    const response = await asyncReturnOrError(deleteAnnotation(uuid), ' Error deleting annotations', res, logger)
 
-        if (response) {
-            res.status(200).send(JSON.stringify(response))
-        }
+    if (response) {
+        res.status(200).send(JSON.stringify(response))
```

Comment:
Is there any particular reason we need to stringify the response btw? I know we're using bodyParser.json() which will convert this back into a json object?

Reply:
removed